src/features/discord/messages/services/command/feature/constants/discord-message-command-feature-names.ts

```diff
@@ -1,13 +1,7 @@
-import { DiscordCommandFirstArgument } from "../../../../classes/commands/arguments/discord-command-first-argument";
 import { DiscordCommandFirstArguments } from "../../../../classes/commands/arguments/discord-command-first-arguments";
 import { DiscordMessageCommandFeatureNameEnum } from "../enums/discord-message-command-feature-name.enum";
+import { DISCORD_MESSAGE_COMMAND_FEATURE_NAME_NOON } from "./discord-message-command-feature-name-noon";
 
 export const DISCORD_MESSAGE_COMMAND_FEATURE_NAMES: DiscordCommandFirstArguments<DiscordMessageCommandFeatureNameEnum> = new DiscordCommandFirstArguments(
-  [
-    new DiscordCommandFirstArgument({
-      description: `Configure the noon message sent at 12AM.`,
-      name: DiscordMessageCommandFeatureNameEnum.NOON,
-      shortcuts: [DiscordMessageCommandFeatureNameEnum.N],
-    }),
-  ]
+  [DISCORD_MESSAGE_COMMAND_FEATURE_NAME_NOON]
```

Comment:
Add coverage (basically for all the constants created without a small, simple coverage